common/logger/src/counters.rs

```diff
@@ -27,15 +27,6 @@ pub static SENT_STRUCT_LOG_COUNT: Lazy<IntCounter> = Lazy::new(|| {
     .unwrap()
 });
 
-/// Count of logs failed to be submitted (but not by the queue)
-pub static DROPPED_STRUCT_LOG_COUNT: Lazy<IntCounter> = Lazy::new(|| {
```

Comment:
this is now ERROR count?

Reply:
This wasn't ever used and it looks like there's already the Error counter.

Reply:
Fun story, it was used, and then cut when I changed the logging system.  It's been broken into queue errors and send errors.

Reply:
Really? I can't seem to find when it was used. Running 
```
$ git log -S DROPPED_STRUCT_LOG_COUNT
```
Only identifies commit 9c09a6394 ([logging] Added metrics around submission of struct logs, 2020-07-28) as it being present in an added/removed line and that commit only defines the counter but never uses it.

Reply:
Okay, then a mistake that I never used it :(